tasks/comment/InfracostComment/index.ts

```diff
@@ -12,12 +11,16 @@ import { promises as fsPromises } from 'fs';
  * @param behavior - post behavior, used by `infracost output` command
  * @return the saved file path
  */
-function generateOutput(path: string, format: string, behavior: string): string {
-  const outputFile = '/tmp/infracost-comment.md';
+async function generateOutput(path: string, format: string, behavior: string): Promise<string> {
+  const outputFile = 'infracost-comment.md';
 
   let pathList: string[];
   try {
     pathList = JSON.parse(path);
+
+    if (typeof(pathList) === 'string') {
+      pathList = [path];
+    }
```

Comment:
`"example*.json"` is parsed without errors, falling back to string here, probably there is a better way to do it.